en/debug/failure_injection.md

```diff
@@ -29,12 +53,12 @@ where:
   - `wrong`: Publish invalid values (that still look reasonable/aren't "garbage").
   - `slow`: Publish at a reduced rate.
   - `delayed`: Publish valid data with a significant delay.
-  - `intermittent`: Publish intermittently. 
+  - `intermittent`: Publish intermittently (sometimes correct, sometimes incorrect). 
```

Comment:
This was correct before.

Reply:
Thanks @julianoes . The thing is, that I rewrote this after looking at Mavsdk docs :-). Publish intermittently is a little ambigous. Does it mean publish right for 20 seconds then stop for 5 seconds, or what?

I guess what I'm saying is that I'm not sure this is all that useful without detail. I'd also want a log entry probably to tell me about each failure, so I could have an easy way to work out where things are failing due to this feature.

Reply:
Oh, and I will revert :-)

Reply:
Intermittently means it's publishing, and then it's not, and then it's publishing again. Basically a lose wire.